RocketApp/app/src/main/java/com/example/rocketapp/BinomialTrial.java

```diff
@@ -7,6 +7,7 @@
 
     private int successNum;
```

Comment:
I think a binomial trial should actually just be a success or a failure rather than counts of successes and failures, right? So I think it may just need a member
boolean isSuccess;
to cover what it represents?

Reply:
The UML diagram I based this code upon had successNum and failureNum in them, the code can be modified to only contain a bool if that's how we want to proceed.

Reply:
Yeah I think it should just be a boolean value to represent its result. We need to change the UML then I think.